src/xcode/ENA/ENA/Source/Services/RecycleBin/RecycleBin.swift

```diff
@@ -0,0 +1,207 @@
+//
```

Comment:
In this file i find three protocols, four enums, one struct and one class. Could we split this up for a better overview?